homeassistant/components/bluetooth_adapters/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""The Bluetooth Adapters integration."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
Isn’t this method also optional?

Reply:
I didn't think it was. I thought we needed one of `async_setup` or `async_setup_entry`

testing now...

Reply:
Blows up without it

```
Logger: homeassistant.setup
Source: setup.py:248
First occurred: 12:55:43 PM (1 occurrences)
Last logged: 12:55:43 PM

Setup failed for bluetooth_adapters: No setup or config entry setup function defined.
```